app/Config/App.php

```diff
@@ -1,5 +1,14 @@
 <?php
 
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
```

Comment:
Why do you add the license notice?

This file is a part of an app, not the framework.
There is no need to claim CodeIgniter Foundation copyright.

Reply:
The reason for adding the license notice is because of the run of command `cs-fix`.

If it is not going to be run on this file and other similar files of cs-fix, then it should be introduced in configuration file .php-cs-fixer, in section **exclude**.

Or at least I thought so.

Reply:
This should not be the case. `app/**/*.php` are linted by `.php-cs-fixer.no-header.php`. If the linter adds the header files, then the command run used the default `.php-cs-fixer.php` with file specified.